Catappult/appcoins-wallet-android
# [ESKL-128/dismiss-notification] - Allows notification to be dismissible

**What does this PR do?**

- Allows notification to be dismissible (through swipe or clear all) after a game ends

**Database changed?**

   No, (I don't know why but I've somehow generated some new schemas?)

**Where should the reviewer start?**

- [ ] skills/src/main/java/cm/aptoide/skills/games/BackgroundGameService.kt

**How should this be manually tested?**

Use our 2048 sample app and play a game (can be sandbox). A notification should appear and should be sticky. End game (you can go back and confirm quit). Notification should now be dismissible (through swipe or clear all).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-128](https://aptoide.atlassian.net/browse/ESKL-128)

**Questions:**

N/A




**Code Review Checklist**
